src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -625,26 +625,71 @@ default int upgradeSSTables(String keyspaceName, boolean excludeCurrentVersion,
     public long getTruncateRpcTimeout();
 
     public void setStreamThroughputMbitPerSec(int value);
+    /**
+     * Please use the new getStreamThroughputMbitPerSecAsDouble method as this one will provide a rounded value
+     * @return stream_throughput_outbound in megabits
```

Comment:
maybe used the deprecated annotation for javadocs here?
```suggestion
     * @return stream_throughput_outbound in megabits
     * @deprecated Use getStreamThroughputMbitPerSecAsDouble instead
```

Reply:
Good call, changed everywhere I had similar deprecation, thanks!